fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -524,6 +526,17 @@ identifierList
     : '(' identifier (',' identifier)* ')'
     ;
 
+identifierOrString
+    : identifier        # identifierOfIdentifierOrString
```

Comment:
There is no need to alias identifer and string separately, the related logic already exists

Reply:
OK, I will remove`visitIdentifierOfIdentifierOrString` and `visitIdentifierOfIdentifierOrString`, then merge the inner logic of these two method to `visitfIdentifierOrString`.